tests/software/ensemblvep/main.nf

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ENSEMBLVEP } from '../../../software/ensemblvep/main.nf' addParams( vep_tag: '104.3.WBcel235', use_cache: false )
```

Comment:
Is it worth testing if `use_cache = true` works as expected? Or is that too much effort?

Reply:
needs to install cache beforehand, so I usually do that locally